src/library/src/test/java/net/aichler/jupiter/tickets/n0001/Ticket0001Test.java

```diff
@@ -22,7 +22,7 @@ public void test() {
 
         assertThat(testRunner.eventHandler().byStatus(Status.Failure), hasSize(1));
         assertThat(testRunner.logger().byLevel("error"), hasItem(
-                containsString("Could not find method [dataProvider]")
+                containsString("Could not find factory method [dataProvider]")
```

Comment:
the error message changed due to 

https://github.com/junit-team/junit5/commit/feeb5bcb6962c7ef997c04ecba474bb17f01480d